src/vs-bicep/Bicep.VSLanguageServerClient.Vsix/Bicep.VSLanguageServerClient.Vsix.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+ <NoWarn>NU1901;NU1902;NU1903;NU1904</NoWarn><!--asdfg do not merge -->
```

Comment:
Was this meant to be included in the PR?

Reply:
No, thanks!